packages/content-conformance/src/rules/__tests__/ensure-valid-link-format.test.ts

```diff
@@ -0,0 +1,157 @@
+import { testRule } from '../../test/utils'
+
+import ensureValidLinkFormat from '../ensure-valid-link-format'
+
+describe('ensure-valid-link-format', () => {
+  test('does nothing for valid developer links', () => {
+    testRule(ensureValidLinkFormat, [
+      {
```

Comment:
🎨 Optional: an additional test case just to be sure it's covered

```suggestion
    testRule(ensureValidLinkFormat, [
      {
        fixture: `[I am a valid link](/non/product/path)`,
        messages: [],
      },
      {
```